doc/source/v2/user/index.rst

```diff
@@ -249,6 +249,32 @@ See the documentation on dask_.
 TODO: This is displaying numpy arrays, not dask. Illustrating dask here might
 require standing up a server.
 
+Explore Metadata
+----------------
+
+Everything recorded at the start of the run is in ``run.metadata['start']``.
+
+.. ipython:: python
+
+    run.metadata['start']
+
+Information only knowable at the end, like the exit status (success, abort,
+fail) is stored in ``run.metadata['stop']``.
+
+.. ipython:: python
+
+    run.metadata['stop']
+
+The v1 API stored metadata about devices involved and their configuration,
+accessed using ``descriptors``, this is roughly equivalent to what is available
```

Comment:
This isn't as precise as I'd prefer. The "baseline" stream contains readings from a different (perhaps overlapping) set of hardware from other streams. From databroker's POV it is "just another stream" and does not necessarily even exist, depending on how the data acquisition system was configured. The "descriptors", which are present in v1 and v2, contain metadata about the columns (dtype, shape, optional units, etc.) and additional one-time only readings from the hardware necessary for interpreting the data in the columns (exposure time, motor velocity setting, etc.). A difference between v1 and v2 is that in v2 descriptor readings are "flattened" into the xarray.Dataset whereas in v1 they had to accessed by a special, separate method. I don't love the flattening, but it serves the purpose of making these readings more discoverable. Perhaps we'll come up with a better way before we declare v2 "stable".

Reply:
I think I get the mapping and made a new commit that hopefully maps v1 to v2 more accurately.

Reply:
This makes it sound a bit like we have gotten rid of descriptors altogether to me rather than re-arrange.  Maybe something like "In v1 you accessed information about devices + configuration by directly accessing the descriptor documents, in v2 this infromation is flattened into ..." ?